src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.Match.SRM.cs

```diff
@@ -1,63 +1,32 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.IO;
-using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions.SRM;
```

Comment:
Later, we should probably globally rename SRM to NonBacktracking.

Reply:
Yes. Namespaces and folders are still on the docket to be cleaned up.  You can see multiple PRs of me incrementally working through the codebase.